Rocket/Rocket/Extensions/StringExtension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  StringExtension.swift
+//  Rocket
+//
+//  Created by Adela Mišicáková on 19.07.2022.
+//
+
+import Foundation
+
+extension String {
+    
+    //MARK: - Formate dateString
+    // - Formates dateString from format yyyy-MM-dd to d.M.yyyy
+    // - Returns formatted dateString
+    
+    func formateDateString () -> String {
```

Comment:
Toto záleží na preferencii programátora, ale obecne sa snažíme funkcie robiť čo najviac prepoužiteľné ako to len ide, takže ja by som pridala možnosť si zvoliť vstupný a výstupný formát, a kľudne im dala default values na `yyyy-MM-dd` a `d.M.yyyy` aby si ich nemusela vždy vyplňovať.